auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -26,24 +26,29 @@
       </div>
 
       <v-row no-gutters id="dashboard-actions" class="mb-n3">
-        <v-col cols="9">
+        <v-flex shrink class="mr-4">
           <!-- Add Existing Name Request or Business -->
           <v-menu
             v-model="addAffiliationDropdown"
           >
-            <template v-slot:activator="{ on }">
-              <v-btn
-                id="add-existing-btn"
-                class="mt-2"
-                color="primary"
-                dark
-                large
-                v-on="on"
-              >
-                <v-icon>mdi-plus</v-icon>
-                <span><strong>Add an Existing Business or Name Request</strong></span>
-                <v-icon class="ml-2 mr-n2">{{ addAffiliationDropdown ? 'mdi-menu-up' : 'mdi-menu-down' }}</v-icon>
-              </v-btn>
+            <template v-slot:activator="{ on: onMenu }">
+              <v-tooltip top content-class="top-tooltip">
+                <template v-slot:activator="{ on: onTooltip }">
+                  <v-btn
+                    id="add-existing-btn"
+                    class="mt-2"
+                    color="primary"
+                    dark large
+                    v-on="{ ...onMenu, ...onTooltip }"
```

Comment:
I think you're the first dev to use v-on like this. What does it do?

Reply:
The multiple activator slots working on v-btn cause a conflict on v-on to apply the tooltip event handler and the menu event handler. So on line 34 and line 36 I assigned them different names  and then ```{ ...onMenu, ...onTooltip }``` stiches them together so they both apply to the btn. [Stack post](https://stackoverflow.com/questions/58075400/using-v-tooltip-inside-v-menu-activator-in-vuetify-2-0) where I got it. It has a good explination.

Reply:
So the tooltip displays on both the dropdown menu icon and the Add button?